dev/TeachingTip/TeachingTip.xaml

```diff
@@ -390,17 +390,19 @@
                                                     </Grid.ColumnDefinitions>
                                                     <Button x:Name="ActionButton"
                                                             HorizontalAlignment="Stretch"
-                                                            Content="{TemplateBinding ActionButtonContent}"
                                                             Style="{TemplateBinding ActionButtonStyle}" 
                                                             Command="{TemplateBinding ActionButtonCommand}"
-                                                            CommandParameter="{TemplateBinding ActionButtonCommandParameter}"/>
+                                                            CommandParameter="{TemplateBinding ActionButtonCommandParameter}">
+                                                            <ContentPresenter TextWrapping="WrapWholeWords" Content="{TemplateBinding ActionButtonContent}"/>
+						    </Button>
```

Comment:
I don't think that those changes are needed for the spanning. Those are about enabling text wrapping on the buttons right?

Reply:
yes, you're correct:)

Reply:
Was just wondering because it was missing from the PR description, but you added it now so it's fine :)